src/StudyApi/Extensions/EndpointRouteBuilderExtensions.cs

```diff
@@ -10,77 +9,67 @@
 using Microsoft.AspNetCore.Routing;
 using Microsoft.Extensions.Diagnostics.HealthChecks;
 
-namespace StudyApi.Extensions
+namespace StudyApi.Extensions;
+
+public static class EndpointRouteBuilderExtensions
 {
-    public static class EndpointRouteBuilderExtensions
+    public static IEndpointRouteBuilder MapCustomHealthCheck(
+        this IEndpointRouteBuilder endpoints,
+        string pattern = "/api/health",
+        string servicesPattern = "/api/health/ready")
     {
-        public static IEndpointRouteBuilder MapCustomHealthCheck(
-            this IEndpointRouteBuilder endpoints,
-            string pattern = "/api/health",
-            string servicesPattern = "/api/health/ready")
+        if (endpoints == null) throw new ArgumentNullException(nameof(endpoints));
```

Comment:
Generally you want to keep braces even for single line statements. Missing braces can lead to scope confusion and subtle bugs.